pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -221,6 +228,8 @@ func updateApiConfig(p interface{}) mf.Transformer {
 			}
 		}
 
+		applyLokiStackConfig(prop, s.LokiStackName, s.LokiStackNamespace)
```

Comment:
you have to apply this config, if the type is `Loki`, otherwise unnecessary

Reply:
I am checking inside this function if we need to apply this or not. I can put that outside also.

Reply:
BTW, check is base on `lokistack` fields on `logsType` field.